test/unit/calculators/minimum_wage_calculator_test.rb

```diff
@@ -161,6 +161,12 @@ class MinimumWageCalculatorTest < ActiveSupport::TestCase
           assert !@calculator.eligible_for_living_wage?
         end
       end
+
+      should 'return false if age is over 25, and date is before 2016-04-01' do
+        @calculator.date = Date.parse('2016-03-30')
+        @calculator.age = 26
+        assert !@calculator.eligible_for_living_wage?
```

Comment:
Not sure about this `assert !@calculator.eligible_for_living_wage?` is this intentional ?